SPECS-EXTENDED/python-kmod/python-kmod.spec

```diff
@@ -1,15 +1,16 @@
 Name:           python-kmod
-License:        LGPLv2+
+License:        LGPL-2.0-or-later
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Summary:        Python module to work with kernel modules
-Version:        0.9
-Release:        28%{?dist}
-URL:            https://github.com/agrover/python-kmod/
-Source0:        https://github.com/downloads/agrover/%{name}/%{name}-%{version}.tar.gz
+Version:        0.9.2
+Release:        7%{?dist}
+URL:            https://github.com/maurizio-lombardi/python-kmod/
+Source0:        https://github.com/downloads/maurizio-lombardi/%{name}/%{name}-%{version}.tar.gz
```

Comment:
Please modify this URL as per the one given in cgmanifest. Without this one of the PR checks is failing. Please take a look.

Reply:
Done.